### PR TITLE
feat: uikit update to new version

### PR DESCRIPTION
**Pull request by Alex-A4**
## Description

copilot:all

# Completed:
- Buttons
- AppBar
- IconButtons
- Colors
- Typographic

# Partially completed
- Inputs

## Type of Change

- [X] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [X] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Review by nesquikm — CHANGES_REQUESTED** [review]

File: lib/feature/add_seed/check_seed_phrase/view/check_seed_phrase_page.dart [hunk 1]
```diff
@@ -35,9 +34,11 @@ class _CheckSeedPhrasePageState extends State<CheckSeedPhrasePage> {
         () => _navigateToPassword(context),
       )..initAnswers(),
       child: Scaffold(
-        appBar: OnboardingAppBar(
+        appBar: DefaultAppBar(
           actions: [
-            CommonButton.ghostNoPadding(
+            CommonButton.ghost(
+              height: 60,
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/feature/add_seed/check_seed_phrase/view/check_seed_phrase_page.dart:40]: Lets wait for ur Padding system, this is not ui-lib part

File: packages/ui_components_lib/example/lib/main.dart [hunk 2]
```diff
@@ -22,7 +22,7 @@ class StoryWidget extends StatelessWidget {
           home: child,
         );
       },
-      initialStory: 'Git log',
+      initialStory: 'Widgets/Buttons',
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/ui_components_lib/example/lib/main.dart:24]: -

File: packages/ui_components_lib/lib/utils.dart [hunk 3]
```diff
@@ -8,3 +8,29 @@ extension StringUtils on String {
   String get overflow =>
       characters.replaceAll(Characters(''), Characters('\u{200B}')).string;
 }
+
+extension ListExtension<T> on Iterable<T> {
+  /// Separate every element of list by [separator].
+  /// Typically used in widgets.
+  List<T> separated(T separator) {
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/ui_components_lib/lib/utils.dart:15]: Lets fix it after Padding system

File: packages/ui_components_lib/pubspec.yaml [hunk 4]
```diff
@@ -24,31 +24,23 @@ flutter:
...
-        - asset: assets/fonts/faktum/Faktum-Regular.ttf
-        - asset: assets/fonts/faktum/Faktum-RegularItalic.ttf
-        - asset: assets/fonts/faktum/Faktum-SemiBold.ttf
-        - asset: assets/fonts/faktum/Faktum-SemiBoldItalic.ttf
-        - asset: assets/fonts/faktum/Faktum-Thin.ttf
-        - asset: assets/fonts/faktum/Faktum-ThinItalic.ttf
+        - asset: assets/fonts/Inter/static/Inter-ExtraBold.ttf
+          weight: 800
+        - asset: assets/fonts/Inter/static/Inter-Black.ttf
+          weight: 900
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply packages/ui_components_lib/pubspec.yaml:45]: -

File: lib/bootstrap.dart [hunk 5]
```diff
@@ -1,10 +1,12 @@
 import 'dart:async';
 
+import 'package:app/app/router/app_route.dart';
 import 'package:app/bootstrap/bootstrap.dart';
 import 'package:app/di/di.dart';
 import 'package:bloc/bloc.dart';
 import 'package:flutter/widgets.dart';
 import 'package:logging/logging.dart';
+import 'package:ui_components_lib/components/common/common.dart';
```
[the remark this answers is not in the document]
Comment (Alex-A4) [reply lib/bootstrap.dart:9]: -

File: lib/feature/add_seed/check_seed_phrase/view/check_seed_phrase_page.dart [hunk 6]
```diff
@@ -37,7 +37,6 @@ class _CheckSeedPhrasePageState extends State<CheckSeedPhrasePage> {
         appBar: DefaultAppBar(
           actions: [
             CommonButton.ghost(
-              height: 60,
               padding: const EdgeInsets.symmetric(horizontal: 16),
```
Comment (nesquikm) [inline lib/feature/add_seed/check_seed_phrase/view/check_seed_phrase_page.dart:40]: Maybe Dimens.medium?

File: lib/feature/add_seed/enter_seed_phrase/view/enter_seed_phrase_view.dart [hunk 7]
```diff
@@ -20,7 +22,7 @@ class EnterSeedPhraseView extends StatelessWidget {
 
     return SafeArea(
       child: Padding(
-        padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 16),
+        padding: const EdgeInsets.all(Dimens.d16),
```
Comment (nesquikm) [inline lib/feature/add_seed/enter_seed_phrase/view/enter_seed_phrase_view.dart:25]: Maybe Dimens.medium?

**Review by nesquikm — APPROVED** [review]

**Pull request merged**